chore: ROOT-204: add LSE-only badges to endpoints in docs

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/18199029975)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/632
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18182736888

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18193684046

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-204'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18193684046

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18193768035

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18195876990

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-204'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18195876990

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18196564525

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18199029975

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-204'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18199029975

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18209412664)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/support.svg

matt-bernstein committed:
Merge branch 'develop' into 'fb-ROOT-204'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18209412664

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/18209795229
